runtime/near-vm/engine/src/universal/builder.rs

```diff
@@ -50,9 +50,8 @@ impl Universal {
     /// Build the `UniversalEngine` for this configuration
     pub fn engine(self) -> UniversalEngine {
         let target = self.target.unwrap_or_default();
-        let pool = self.pool.unwrap_or_else(|| {
-            super::LimitedMemoryPool::new(1, 10 * rustix::param::page_size()).unwrap()
-        });
+        let pool =
+            self.pool.unwrap_or_else(|| panic!("Universal::code_memory_pool was not set up!"));
```

Comment:
Why not `.expect("...")`?

Reply:
It doesn’t read right! The text would need to be negated, but that then doesn’t make sense to the person seeing this message.